solana/programs/02_hello_token/Cargo.toml

```diff
@@ -19,6 +19,6 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = { version="0.28.0", features = ["init-if-needed"]}
+anchor-lang = { version = "0.28.0", features = ["init-if-needed"]}
 anchor-spl = "0.28.0"
-wormhole-anchor-sdk = { path = "../../modules/wormhole-anchor-sdk", default-features = false, features = ["token-bridge"]}
+wormhole-anchor-sdk = { version = "0.1.0-alpha.1", default-features = false, features = ["token-bridge"]}
```

Comment:
since we're specifying the version here instead of a path, can we delete the sdk from this repo?

Reply:
Would like to do that, but we need to find a good spot for it. I'm all ears :ear: